sequins.go

```diff
@@ -216,6 +221,12 @@ func (s *sequins) initCluster() error {
 		return err
 	}
 
+	flapNotify := zkWatcher.SetFlapThreshold(flapMax, flapDuration)
+	go func() {
+		<-flapNotify
+		panic("Dying due to ZK flapping")
```

Comment:
I really don't like panics as they are hard to parse in splunk, can we change this to a `log.Fatal`.  We don't really care about the stack trace here do we?